src/OpenTelemetry.Instrumentation.GrpcNetClient/README.md

```diff
@@ -1,17 +1,53 @@
-# OpenTelemetry.Instrumentation.GrpcNetClient
+# Grpc.Net.Client Instrumentation for OpenTelemetry
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Instrumentation.GrpcNetClient.svg)](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.GrpcNetClient)
 
-Automatically instruments the outgoing requests from
-[Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client).
+This is an [Instrumentation Library](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/glossary.md#instrumentation-library)
+which instruments [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client)
+and collects telemetry about outgoing gRPC requests.
 
-## Installation
+## Steps to enable OpenTelemetry.Instrumentation.GrpcNetClient
+
+### Step 1: Install Package
+
+Add a reference to the
+[`OpenTelemetry.Instrumentation.GrpcNetClient`](https://www.nuget.org/packages/opentelemetry.instrumentation.grpcnetclient)
+package. Also, add any other instrumentations & exporters you will need.
 
 ```shell
 dotnet add package OpenTelemetry.Instrumentation.GrpcNetClient
 ```
 
+### Step 2: Enable Grpc.Net.Client Instrumentation at application startup
+
+Grpc.Net.Client instrumentation must be enabled in your code. The following
+example demonstrates how to do this for an ASP.NET Core application. The
```

Comment:
Wasn't sure what kind of application made the most sense to demonstrate. I can see ASP.NET Core being a common scenario though perhaps demonstrating a console application is useful too?

Reply:
the key here is show the extension method on TracerBuilder. Where it should be run , ofcourse, depends on the application - in console, its one place. In asp.net core is typically Startup.cs.

Could we say, do this in the application startup. For example, if you are using an Asp.Net Core application, then this is typically the COnfigureServices of you Startup.cs class.. 

(something of the above tune)

Reply:
Updated this section. Simplified with simple console app example, but also worked in your verbiage.